tasks/src/text-generation/data.ts

```diff
@@ -38,25 +38,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["text-generation"],
 	metrics:   [
 		{
-			description: "Cross Entropy is a metric that calculates the difference between two probability distributions. Each probability distributions is the distribution of predicted words.",
+			description: "The Cross Entropy metric calculates the difference between two probability distributions. Each probability distribution is the distribution of predicted words.",
 			id:          "Cross Entropy",
 		},
 		{
-			description: "Perplexity is the exponential of the cross-entropy loss. It evaluates the probabilities assigned to the next word by the model. Lower perplexity indicates good performance.",
+			description: "The Perplexity metric is the exponential of the cross-entropy loss. It evaluates the probabilities assigned to the next word by the model. Lower perplexity indicates better performance.",
 			id:          "Perplexity",
 		},
 	],
 	models: [
 		{
-			description: "The model from OpenAI that helped usher in the Transformer revolution.",
+			description: "The OpenAI model that helped usher in the Transformer revolution.",
 			id:          "gpt2",
 		},
 		{
 			description: "A special Transformer model that can generate high-quality text for various tasks.",
 			id:          "bigscience/T0pp",
 		},
 	],
-	summary:      "Text generation is the task of...well generating text! These models can complete incomplete text or asked to perform a task such as paraphrasing. This depends on the model’s training objective (but don’t worry, we will cover both of them!).  ",
+	summary:      "Generating text is the task of producing new text. These models can, for example, fill in incomplete text or paraphrase.",
```

Comment:
We like to keep things playful :') I liked the old one more

Reply:
Sounds good to me, however, I am concerned that I have only observed this tone in this Summary. So I thought it would be good to make them uniform. WDYT? I can make the change, no problem.